config/initializers/session_store.rb

```diff
@@ -1,9 +1,16 @@
 service_url = ENV['SERVICE_URL']
 SERVICE_DOMAIN = (service_url) ? URI.parse(service_url).host : nil
+timeout_time = ENV['TIMEOUT_TIME']
+
+if Rails.env.developement?
+  timeout_time
```

Comment:
This line does nothing; should it be trying to convert `timeout_time` into minutes?